src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.Tests/CallSiteTests.cs

```diff
@@ -174,9 +174,10 @@ public void BuildExpressionAddsDisposableCaptureForDisposableFactoryServices(Ser
         }
 
         [Theory]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/52863")]
```

Comment:
This PR closes the linked issue.

Reply:
yes but I'm trying to first add this one liner to confirm no other tests failing. 

Meanwhile trying to figure out if this failure and `BuildExpressionElidesDisposableCaptureForEnumerableServices` for scoped failure is expected...